.github/workflows/flowzone.yml

```
@@ -0,0 +1,18 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+      - '20[0-9][0-9].[0-1]?[1470].x'
+
+jobs:
+  flowzone:
+    name: Flowzone
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
+    secrets:
+      FLOWZONE_TOKEN: ${{ secrets.FLOWZONE_TOKEN }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
```

Since we moved to enterprise we can now use:
```suggestion
    secrets: inherit
```
to simplify secrets handling

---

thanks @Page- I was using inherit for the `build_and_deploy` workflow before being in enterprise, what's the relation with enterprise?

---

You can only inherit secrets for workflows that are in the same organization/enterprise, so before we became an enterprise flowzone was in a different org and hence you couldn't inherit secrets